### PR TITLE
gh-95724: change the taskgroup permission notice to reflect who the permission was given to

### PR DESCRIPTION
**Pull request by graingert**
Previously the permission notice was relative to the current file, which made it unclear if people copying taskgroups.py would also need to gain permission from edgedb

* Issue: gh-95724

### CODE REVIEW

File: Lib/asyncio/taskgroups.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Adapted with permission from the EdgeDB project.
+# Licensed to PSF from the EdgeDB project.
```
Comment (gvanrossum) [inline Lib/asyncio/taskgroups.py:1]: This would seem to imply that it is unchanged from the EdgeDb version, which isn't the case at all.

**Comment by erlend-aasland** [comment]
Please update the PR title to accurately reflect the proposed change.

File: Lib/asyncio/taskgroups.py (hunk 1 above)
Comment (graingert) [reply Lib/asyncio/taskgroups.py:1]: Hmm, I certainly don't want to imply that either

**Comment by gvanrossum** [comment]
@graingert This also needs to be done to test_taskgroups.py

**Comment by gvanrossum** [comment]
@graingert Please see https://github.com/python/cpython/pull/95847 for Yury's alternative PR.

**Pull request closed without merging**